<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditarCalculo</class>
 <widget class="QMainWindow" name="EditarCalculo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoreBeaches    Editar Cálculo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>FormatFactoryAgregar un título.ico</normaloff>FormatFactoryAgregar un título.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 74, 116);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_NuevoCalculo">
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color: rgb(201, 230, 255);
}
QLabel{
background-color: #000000ff;
color::rgb(0, 74, 116);
border:8px solid#000000ff;
}



QLineEdit{
background-color: rgb(255, 255, 255);
}


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_NuevoCalculo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>641</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 74, 116);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Editar Cálculo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_DensidadArena">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Densidad de Arena (ρs)*</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_densidadArena">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>150</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_DensidadMar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>210</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Densidad del Mar (ρ)*</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_DensidadMar">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>210</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_CoeficientePorocidad">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>270</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Coeficiente de Porocidad (n)*</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_CoeficientePorocidad">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>270</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_ALtura">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>330</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Altura (Hb)*</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_altura">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>330</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_AnguloR">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>150</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ángulo de Rompiente (α)*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_IndiceR">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>210</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Índice de Rompiente(k)*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Ubicacion">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>270</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ubicación*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SGCTS_GeoCuba">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>0</y>
         <width>221</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #000000ff;

color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;MoreBeaches&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Ubicacion">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>270</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_IndiceRompiente">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>210</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_AnguloRompiente">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>150</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QPushButton" name="but_NuevoEditar">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>460</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(0, 0, 0);
	background-color: rgb(25, 255, 98);
border-radius:8px;
}

QPushButton:hover{
background-color: rgb(47, 255, 158);
border-top-left-radius:12px;
border-bottom-left-radius:12px;

}</string>
       </property>
       <property name="text">
        <string>Nuevo</string>
       </property>
      </widget>
      <widget class="QPushButton" name="butCancelarEditar">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>460</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255, 255, 255);
	background-color: rgb(221, 1, 5);
border-radius:8px;
}

QPushButton:hover{
background-color: rgb(255, 120, 129);
border-top-left-radius:12px;
border-bottom-left-radius:12px;

}</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="but_ActualizarEditar">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>460</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(11, 15, 255);
color: rgb(0, 0, 0);
border-radius:8px;
}

QPushButton:hover{
background-color: rgb(16, 132, 255);
border-top-left-radius:12px;
border-bottom-left-radius:12px;

}</string>
       </property>
       <property name="text">
        <string>Actualizar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Error">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>390</y>
         <width>641</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 1, 5);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Label Error&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 1, 5);</string>
       </property>
       <property name="text">
        <string>Los campos con (*)  son obligatorios</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>0</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Agregar_un_título-removebg-preview.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
